api.go

```diff
@@ -3,7 +3,7 @@ package luno
 import (
 	"context"
 
-	"github.com/luno/luno-go/decimal"
```

Comment:
Try not to change module names when creating PR's to public repos from your forks.

Reply:
You can use a replace directive in your go.mod file to be able to leave these lines unmodified, yet still use your local module. See [this SO answer](https://stackoverflow.com/questions/14323872/using-forked-package-import-in-go) for an example